src/LocalistProcessor.php

```diff
@@ -319,27 +370,27 @@ public function process_url_pull($search_field_name,$url) {
 
         //recursive call until we hit localist_page_count limit if configured, max of 3 pages
         $next_page_url = $this->create_localist_url($current_page + 1);
-        $this->process_url_pull($search_field_name,$next_page_url);
+        $this->process_url_pull($search_field_name, $next_page_url);
       }
     }
     catch (RequestException $e) {
-      \Drupal::logger('localist_pull')->error($e->getMessage());
+      \Drupal::logger('localist_pull')->notice("exception");
```

Comment:
@judaw Does this change affect dblog message content, is `$e` no longer output in a log message, or is that not what's going on here?

Reply:
this was a change on the other PR not sure why it was done originally but it will just be less destructive throwing a notice. Looks like a try catch around the request so probably just handling when requests don't come back from localist. It can get changed back or stay this way I don't have a strong opinion

Reply:
Yeah its static. It could go back to $e->getMessage(); or something like "Could not retrieve data from localist"

Reply:
Ok I changed it back to `$e->getMessage()` for now, though, I guess I don't know what is in `$e` ..... I'm guessing `$e` is a local variable used or generated by `RequestException` ??  I'm fine with whatever message makes sense, I just wanted something more meaningful than "exception".  It used to be `$e->getMessage()`, but it's not like I know what that actually outputs.

Thoughts/feelings? -- and, is there anything that should be tested on IT@C with this log message change?